src/ape/utils.py

```diff
@@ -130,10 +164,35 @@ def deep_merge(dict1, dict2):
 
 
 def expand_environment_variables(contents: str) -> str:
+    """
+    Used to expand the enviroment variable given the path.
+
+    Args:
+        contents (str): A path-like object representing a file system path.
```

Comment:
This does necessarily have to be a path.
This is just a `str` with un-expanded env vars in it